.github/workflows/update-man-pages.yml

```diff
@@ -0,0 +1,39 @@
+name: Update man-pages workflow
+on:
```

Comment:
if we change this to be a manually-triggered workflow, then the username question for the commit that's created has an easy answer - the name/email of the user that triggered the action